test/test_condensed_downloads.py

```diff
@@ -0,0 +1,36 @@
+# Copyright (C) 2021 Intel Corporation
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+"""
+Condensed Downloads Test
+"""
+import importlib
+import pytest
+import subprocess
+from test.test_data import __all__ as all_test_name
+
+# Test to check condensed files are commited according to the package test data of checkers
+def test_condensed_downloads():
+    test_data = list(
+        map(lambda x: importlib.import_module(f"test.test_data.{x}"), all_test_name)
+    )
+
+    package_names = []
+    package_test_datas = map(lambda x: x.package_test_data, test_data)
```

Comment:
I really hate having `package_test_datas` and `package_test_data` as separate variables, although I guess it's clear what they do.  Could we change the first one to `package_test_data_list` or something else that's more than a one-character difference but reflects the actual data structure correctly?

Reply:
I've solved that. And yep it could be confusing for people who read the code :sweat_smile: .